pandas/tests/copy_view/test_indexing.py

```diff
@@ -772,6 +772,34 @@ def test_column_as_series_set_with_upcast(using_copy_on_write, using_array_manag
         tm.assert_frame_equal(df, df_orig)
 
 
+def test_column_as_series_no_item_cache(using_copy_on_write, using_array_manager):
+    # Case: selecting a single column (which now also uses Copy-on-Write to protect
+    # the view) should always give a new object (i.e. not make use of a cache)
+    df = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [0.1, 0.2, 0.3]})
+    df_orig = df.copy()
+
+    s1 = df["a"]
```

Comment:
Is it worth testing the other indexing methods that will return a columns as a Series (loc/iloc)?

Reply:
Good call, `loc` seems to use the same caching, so good to cover that in the test as well. Updated.